myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -92,6 +92,7 @@
    */
   public static final Boolean DEFAULT_HA_ENABLED = false;
 
+  public static final Boolean DEFAULT_SECURITY_ENABLED = false;
```

Comment:
DEFAULT_AUTHENTICATION_ENABLED or DEFAULT_AUTH_ENABLED? "Security" is too vague, since encryption, authorization, and isolation are all parts of "security".